thought.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose'
+
+export const Thought = mongoose.model(
```

Comment:
Cool that you made a separate file for the mongoose model! Makes it more tidy 👍🏼👍🏼☺️